src/main/java/com/vecna/maven/db/mysql/MysqlStrategy.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2014 Vecna Technologies, Inc.
```

Comment:
> The copyright notice on a work establishes a claim to copyright. The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date and that means if somebody has copied the work in the meantime and they claim its theirs on the ground that their publishing the copy was before your claim, then it will be difficult to establish who is the originator of the work.

from https://stackoverflow.com/a/2391555

Seems copyright should stay old.  There is some other stuff the post says, but I don't think that applies here.